### ISSUE TITLE
Modules with updates

### ISSUE THREAD

**Issue by dromer** [DISTRHO/Cardinal#638]
### Description

The following modules have updates that we can integrate.

Bugfixes:
- AudibleInstruments (CardinalModules)
- Bogaudio
- Bidoo (CardinalModules)
- Valley (CardinalModules)
- ZZC
- ChowDSP (CardinalModules)
- MockbaModular (tiny fix)
- HamptonHarmonics
- ML_modules
- Orbits
- AaronStatic (tiny fix)
- forsitan-modulare
- h4h4-modules

Features:
- JW-Modules (dark panels)
- ImpromptuModular (features + fixes)
- BaconPlugs (features + fixes)
- sonusmoduler (features + fixes)
- LilacLoop (expander + model rename)
- voxglitch (we are behind a _lot_ with this one. some major UI changes and other features)
- Fundamental (bunch of new modules and other features. requires custom panels)
- Meander (mostly fixes and two new modules)
- alefsbits (couple fixes and many new modules)
- EnigmaCurry (new module)

Minor:
- MindMeldModular (some fixes, but not released yet)
- LyraeModules (internal renaming)
- Little-Utils (some changes to port behavior)
- nonlinearcircuits (small refactor and svg renames/changes)

**Comment by falkTX** [comment]
voxglitch is finally updated, I looked at Fundamental but dont quite think the new modules are worth it

**Comment by dromer** [comment]
I think we should try to have all fundamental eventually because users really expect those to be there from the start, but maybe not for this `24.09` release as it requires panels etc.

**Comment by rl2939** [comment]
You might want to add Befaco to the list, because v2.8.0 adds two new modules (Bypass and Bandit).

It might also be worth converting this list of bullet points into a task list so that the issue tracker itself can display how many tasks are left in the issue.
```
To create a task list, preface list items with a hyphen and space followed by [ ]. To mark a task as complete, use [x].

- [x] #739
- [ ] https://github.com/octo-org/octo-repo/issues/740
- [ ] Add delight to the experience when all tasks are complete :tada:
```
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists

**Comment by dromer** [comment]
> You might want to add Befaco to the list, because v2.8.0 adds two new modules (Bypass and Bandit).
This list was created some time ago, so it is of course never fully up to date. By the time we are done with it some of the modules have likely updated again.

> It might also be worth converting this list of bullet points into a task list so that the issue tracker itself can display how many tasks are left in the issue.
Good idea, have done that just now. Then when modules need updating again we can "un-check" them so they become active tasks again ;)

**Comment by dromer** [comment]
>  This is significant

Not really. We keep these forks around in https://github.com/CardinalModules organization.

We do try to push Cardinal specific changes upstream, but as long as these devs don't merge our changes we will have to keep our own forks around.

**Comment by dromer** [comment]
There's updates for Autinn: https://community.vcvrack.com/t/autinn-plugin/25486

**Comment by acsl9r-tech** [comment]
Would it be possible to pull in a newer version of [impromptu modules](https://github.com/MarcBoule/ImpromptuModular)? The version we are currently using doesn't include stuff like NoteEcho/NoteLoop/NoteFilter (https://github.com/MarcBoule/ImpromptuModular?tab=readme-ov-file#note-echo-loop-filter). Thanks!

**Comment by dromer** [comment]
Everything is possible, but someone will need to do the work. And as mentioned in the overview some modules were removed so we have to fork and add them back for backwards compatibility.

(I have to double-check, but I think they didn't just deactivate the modules and really removed the code]

**Comment by rl2939** [comment]
Befaco needs to be updated again. The new update adds a new `lib` folder (a new module relies on this code) that does not appear to build with the rest of Cardinal. (At least, I am having trouble building it.)
